src/hooks/useTracking.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  useAnalyticsContext,
+  AnalyticsEventAction,
+  AnalyticsEventCategory,
+  AnalyticsEventLabel,
+  AnalyticsEventType,
+} from '~context/AnalyticsContext';
+
+const useTracking = (
```

Comment:
hope it's not too nitpicky, but I don't think we need this wrapper. We pass it the same props as we do just to the `trackEvent` function. We can just always use `trackEvent` which works nicely